integration-tests/rita.py

```diff
@@ -693,7 +693,9 @@ def test_endpoints_all(self):
                     pprint(neighbors)
                 else:
                     print(result.stdout.read().decode('utf-8'))
-            except json.JSONDecodeError as e:
+            except ValueError as e:
+                if VERBOSE:
+                    print('Unable to decode JSON {!r}: {!r}'.format(result, e))
```

Comment:
Unnecessary, the `assert_test` below will print `<red plus> Decoding the JSON Failed`